specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json

```diff
@@ -1872,6 +2113,29 @@
         }
       }
     },
+    "HierarchyHit": {
+      "description": "The hierarchy node which contains the instances matching the query based on the input. May be empty or null.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the hierarchy node. May be empty, cannot be null.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "cumulativeInstanceCount": {
+          "description": "Total number of instances that belong to this node and it's subtrees matching the query.",
```

Comment:
You can add "subtrees" to https://github.com/Azure/azure-rest-api-specs/blob/master/custom-words.txt if it's the right word

Reply:
I have added it in the branch.